### PR TITLE
Update docs docker base image from slim-buster to slim-bullseye

### PR DESCRIPTION
**Pull request by daveqnet**
# Purpose

* Update the base image used for the docs image to match the fidesops base image (see https://github.com/ethyca/fidesops/pull/928)

# Changes
- Changed base image from Debian slim-buster to slim-bullseye

# Checklist
- [x] Update [`CHANGELOG.md`](https://github.com/ethyca/fidesops/blob/main/CHANGELOG.md) file
  - [x] Merge in main so the most recent `CHANGELOG.md` file is being appended to
  - [x] Add description within the `Unreleased` section in an appropriate category. Add a new category from the list at the top of the file if the needed one isn't already there.
  - [x] Add a link to this PR at the end of the description with the PR number as the text. example: [#1](https://github.com/ethyca/fidesops/pull/1)
- [ ] Applicable documentation updated (guides, quickstart, postman collections, tutorial, fidesdemo, [database diagram](https://github.com/ethyca/fidesops/blob/main/docs/fidesops/docs/development/update_erd_diagram.md).
- If docs updated (select one):
  - [ ] documentation complete, or draft/outline provided (tag docs-team to complete/review on this branch)
  - [ ] documentation issue created (tag docs-team to complete issue separately)
- [ ] Good unit test/integration test coverage
- [ ] This PR contains a DB migration. If checked, the reviewer should confirm with the author that the [down_revision correctly references the previous migration](https://ethyca.github.io/fidesops/development/contributing_details/#alembic-migrations) before merging
- [ ] The `Run Unsafe PR Checks` label has been applied, and checks have passed, if this PR touches any external services

# Ticket

n/a

# Additional info

* I've confirmed that `docker-compose up docs` completes successfully and serves the docs correctly locally.

### CODE REVIEW

**Comment by daveqnet** [comment]
This is a tiny change, just trying to harmonize the base images used in Dockerfiles across the repo.

I haven't updated the changelog since I'm not sure if this is something that should be done at the end or beginning of the PR review process, but let me know if I need to (more than happy to).

File: docs/fidesops/Dockerfile.docs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.8-slim-buster
+FROM python:3.8-slim-bullseye
```
Comment (sanders41) [inline docs/fidesops/Dockerfile.docs:1]: Let's go ahead and harmonize the Python version also.

```suggestion
FROM python:3.9-slim-bullseye
```
```

**Review by sanders41 — COMMENTED** [review]
Thanks for catching this! You can go ahead and update the CHANGELOG now.

**Comment by daveqnet** [comment]
Much obliged, @sanders41, changes made as requested. 🙂

**Review by seanpreston — APPROVED** [review]

**Pull request merged**